frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -392,12 +423,49 @@ const SECTIONS = updateSectionsWithPlugins({
         widget: EmbeddedQuestionListing,
         getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
       },
+      {
+        widget: EmbeddingCustomizationInfo,
```

Comment:
Render extra content for OSS users mentioning paid plans.
![image](https://user-images.githubusercontent.com/1937582/176175412-8f4849e5-e4bb-4c21-b679-c9197b6b78fa.png)